### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by krise3**
@harrisonywu Here's the setup portion of the work I did today, please review when you have the chance

Changelog:
- Install dependencies
- Create server app file
- Create public resource folder

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "proxy_photos",
+  "version": "1.0.0",
+  "description": "Package file for item detail page proxy server",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (harrisonywu) [inline package.json:8]: script to run your server?

File: server/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const port = 3000;
+
+app.use(express.static(path.join(__dirname, '../public')));
+
+app.listen(port, () => console.log('[Server] Listening on port 3000'));
```
Comment (harrisonywu) [inline server/index.js]: add a newline here

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const path = require('path');
+
+const app = express();
+
+const port = 3000;
+
+app.use(express.static(path.join(__dirname, '../public')));
```
Comment (harrisonywu) [inline server/index.js:8]: you probably want to load the static files at '/location/:locationId' instead of just at '/'.

**Review by harrisonywu — APPROVED** [review]
just a few comments, and then you can merge!

File: server/index.js (hunk 3 above)
Comment (krise3) [reply server/index.js:8]: This is fixed in the next branch that I will merge

File: package.json (hunk 1 above)
Comment (krise3) [reply package.json:8]: Totally forgot about it, thanks for the catch!

File: server/index.js (hunk 2 above)
Comment (krise3) [reply server/index.js]: Good catch! I totally forgot to set up ESLint here, going to do that next

**Pull request merged**